semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -0,0 +1,550 @@
+# SemanticDB Specification, Version 3.0.0
+
+  * [Motivation](#motivation)
+  * [Overview](#overview)
+  * [Data Model](#datamodel)
+    * [TextDocument](#textdocument)
+    * [Range](#range)
+    * [Symbol](#symbol)
+    * [SymbolInformation](#symbolinformation)
+    * [SymbolOccurrence](#symboloccurrence)
+    * [Diagnostic](#diagnostic)
+    * [Synthetic](#synthetic)
+  * [Data Schemas](#dataschemas)
+    * [Protobuf](#protobuf)
+    * [JSON](#json)
+    * [SQL](#sql)
+  * [Changelog](#changelog)
+    * [3.0.0](#300)
+
+## Motivation
+
+Nowadays, there is a clear trend towards standards for communication between developer tools.
+Language Server Protocol (LSP) [\[2\]][2], a protocol that connects programming
+language implementations and text editors, has gained strong industrial support
+and at the time of writing has implementations for over 25 programming languages (TODO).
```

Comment:
I left some TODO markers scattered in the specification. While you guys review the pullrequest, I'll finish writing the text.